example-get-started-experiments/code/notebooks/TrainSegModel.ipynb

```diff
@@ -41,7 +22,10 @@
     "from fastai.vision.all import (Resize, SegmentationDataLoaders,\n",
     "                               imagenet_stats, models, unet_learner)\n",
     "from ruamel.yaml import YAML\n",
-    "from PIL import Image"
+    "from PIL import Image\n",
+    "\n",
+    "os.chdir(\"..\")\n",
```

Comment:
By changing to the root dir at the start, we end up with similar behavior to the pipeline and the notebook writes to `dvc.yaml` there.